doc/sphinx/proof-engine/ltac2.rst

```diff
@@ -279,6 +279,7 @@ There is dedicated syntax for list and array literals.
    | ( @ltac2_expr : @ltac2_type )
    | ()
    | [ {*; @ltac2_expr5 } ]
+   | %{ @ltac2_expr0 with {? {+ @tac2rec_fieldexpr } {? ; } } %}
```

Comment:
Would you add a short description of the feature below?  While it's OK to mention that it's similar to OCaml, it shouldn't require knowing the OCaml feature since (I believe) most users don't use OCaml.

Reply:
Also, how about a changelog entry for this?